### DISCUSSION TITLE
Crashes while finding Albums

### DISCUSSION THREAD

**Discussion by feerles85** [TheLastGimbus/GooglePhotosTakeoutHelper#302] [I NEED HELP] [answered]
the Tool/script keep Crashing while or after finding the Albums
(Finding albums (this may take some time, dont worry :) ...)
I have WIndows 11
It is around 200GB Data and i used two seperate SSDs

![grafik](https://github.com/TheLastGimbus/GooglePhotosTakeoutHelper/assets/165081518/f5c2d95f-15f5-4b60-bed6-9783d98b8448)

**Comment by TheLastGimbus** [comment]
Does it crash, or freeze? If the second, then it may just be doing a lot of calculations, and needs time for that...

**Reply by feerles85** [reply]
it crashes, (the window disappears)

**Reply by TheLastGimbus** [reply]
Can you tell the ram usage just before it happens?

Also, I probably won't figure out the solution right now, soooo your best bet is to skip the albums :(

Also, "i use two separate SSDs" - there is a known issue that gpth doesn't like that - #251 - try to put everything on one disk. Gpth moves files by default, so it will use the same space, not 2x (i think this is probably the real issue here)

**Reply by feerles85** [reply, marked as answer]
The RAM varies between 0.2% and 0.5% but it also had a short peak of 0.6%
but now it worked, I used this time same SSD for the Progress
